src/platforms/common/configuration/options.mdx

```diff
@@ -509,4 +509,58 @@ You should follow the [guide to native initialization](/platforms/react-native/m
 
 </ConfigKey>
 
+<ConfigKey name="enableNativeCrashHandling" supported={["react-native", "javascript.capacitor"]}>
+
+Set this boolean to `false` to disable the hard crash handling from the native layer. Doing so means it won't capture events for hard crashes on Android and iOS if the error was caused by native code.
```

Comment:
what is "it" in the second sentence? Let's spell that out just to be super clear. 

```suggestion
Set this boolean to `false` to disable hard crash handling from the native layer. Doing so means it won't capture events for hard crashes on Android and iOS if the error was caused by native code.
```

Reply:
`it` means the Sentry RN SDK.
https://github.com/getsentry/sentry-docs/pull/4828/commits/99ed44716e6ab09a1c3adc5988e3d5072071e3b8